rsqueakvm/model/base.py

```diff
@@ -273,7 +262,6 @@ class W_AbstractObjectWithClassReference(W_AbstractObjectWithIdentityHash):
     """Objects with arbitrary class (ie not CompiledMethod, SmallInteger or
     Float)."""
     _attrs_ = ['w_class']
-    _immutable_fields_ = ['w_class?']
```

Comment:
The DSA algorithm uses changeClass: on LargeInteger<->ByteArray, so this really isn't quasi-immutable in that loop. I'm hoping this doesn't hurt too much